### PR TITLE
Edited the adaptive timeout requirement.

### PR DESCRIPTION
**Pull request by m1cm1c**

### CODE REVIEW

File: doc/Requirements Specification/Parts/Functional Requirements.lyx [hunk 1]
```diff
@@ -308,13 +308,15 @@ name "/F60/"
...
- Beagle stops measurements by an adaptive timeout when enabled.
- This means that it stops measurements if the software fails to respond
- within a certain period of time.
- This timeout is adapted based on previous runs, so that measurements of
- code sections that were fast in earlier measurements are aborted sooner
- if they fail to respond until then than measurements of sections known
- to be slow.
+ Beagle provides a fuctionality to stop measurements by an adaptive timeout
+ when enabled.
+ This means that it aborts a measurement when the latter execeeds a certain
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Functional Requirements.lyx]: I’d prefer “the latter”- > “it” for simplicity. I think it’s absolutely clear what the object is.

Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Functional Requirements.lyx]: Fixed it.

**Comment by flyfl** [comment]
lgtm

**Comment by jGleitz** [comment]
lgtm

**Pull request merged**